bauro-ansible/host_vars/localhost.yml

```diff
@@ -0,0 +1,6 @@
+---
+# file: host_vars/localhost
+
+ansible_connection: local
+
+names_file: ../example_names
```

Comment:
Can you make the this a command line argument for `run.sh`, so we can use a given file?